pkg/config/generate_test.go

```diff
@@ -18,7 +18,10 @@ func TestGenerate(t *testing.T) {
 		},
 		//TODO "test generate doesn't overwrite": {},
 	}
-	var pluginConfig api.PluginConfig
+	tc := api.TestConfig{}
+	pluginConfig := api.PluginConfig{
+		TestConfig: &tc,
```

Comment:
revert this, the plugin has to work with TestConfig: nil, hence so should the test